mnc/control.py

```diff
@@ -173,27 +173,26 @@ def start_fengine(self, snap2names=None, initialize=False, program=False, force=
         ec = snap2_feng_etcd_client.Snap2FengineEtcdControl()
         is_programmed = ec.send_command(0, 'fpga', 'is_programmed', n_response_expected=11)
 
-        if program:
-            if (not all(is_programmed.values()) or force):
-                resp = ec.send_command(0, 'feng', 'program', timeout=120, n_response_expected=1, kwargs={'fpgfile': FPG_FILE})
-                if not resp:
-                    raise RuntimeError('Programming failed. Check f-engine etcd service logs as pipeline@calim.')
-                logger.info("Waiting 120 seconds (ignore error messages above)")
-                time.sleep(120)
-
-                is_programmed = ec.send_command(0, 'fpga', 'is_programmed', n_response_expected=11)
-                if not all(is_programmed.values()):
-                    raise RuntimeError('Programming of other snaps failed. Check f-engine service logs as pipeline@calim.')
-                else:
-                    logger.info("All snaps programmed")
-                resp = ec.send_command(0, 'controller', 'stop_poll_stats_loop')
-            else:
-                logger.info('All snaps already programmed.')
+#        if program:
```

Comment:
why's everything commented out?

Reply:
I wanted to simplify to logic that works robustly and I thought this was a good start. Could we try working these lines back in with a future PR?
I messed up by pushing more commits into this PR, so it would be nice to close this one.

Reply:
Go ahead